tgui/packages/tgui/interfaces/Vending.tsx

```diff
@@ -219,7 +220,7 @@ const ProductDisplay = (props: {
           })
           .map((product) => (
             <VendingRow
-              key={product.name}
+              key={product.path || product.name}
```

Comment:
```suggestion
              key={product.name}
```
There is no need to use  name now since path is  better and guaranteed to be unique. 

Also where is path being passed from the back end? Was it always there?

Reply:
I see, I don't feel comfortable using a var that is sometimes null and sometimes isnt best get expert opinion on this.

@jlsnow301 thoughts?

Reply:
They only need to be unique, even index is fine here

Reply:
> I will have you know I am doing this without breakpoints,

I feel your pain, debugger stopped working after the 515 update.

Hopefully https://github.com/tgstation/tgstation/issues/80008#issuecomment-1831471550 should help you with that.